targets/targets.json

```diff
@@ -1552,13 +1512,9 @@
             "lpticker_lptim": {
                 "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
                 "value": 1
-            },
-            "lpticker_delay_ticks": {
-                "help": "In case of lpticker_lptim=1, set lpticker_delay_ticks=3",
-                "value": 3,
-                "macro_name": "LPTICKER_DELAY_TICKS"
             }
         },
+        "overrides": {"lpticker_delay_ticks": 4},
```

Comment:
Why are 4 needed for the L series?

Reply:
Default value 1 seems not enough for low speed targets.
4 is the experimental result to pass speed test.

Reply:
Is there anything in the reference manual for these devices which state the number of cycles which must elapse before writing to the match register?